<ui version="4.0" >
 <class>restoreForm</class>
 <widget class="QWidget" name="restoreForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="0" >
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTreeWidget" name="directoryWidget" >
      <property name="sizePolicy" >
       <sizepolicy>
        <hsizetype>5</hsizetype>
        <vsizetype>7</vsizetype>
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize" >
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize" >
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="sizeIncrement" >
       <size>
        <width>1</width>
        <height>0</height>
       </size>
      </property>
      <property name="baseSize" >
       <size>
        <width>50</width>
        <height>0</height>
       </size>
      </property>
      <property name="alternatingRowColors" >
       <bool>true</bool>
      </property>
      <property name="columnCount" >
       <number>1</number>
      </property>
      <column>
       <property name="text" >
        <string>1</string>
       </property>
      </column>
     </widget>
     <widget class="QTreeWidget" name="fileWidget" >
      <property name="sizePolicy" >
       <sizepolicy>
        <hsizetype>7</hsizetype>
        <vsizetype>5</vsizetype>
        <horstretch>144</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize" >
       <size>
        <width>400</width>
        <height>0</height>
       </size>
      </property>
      <property name="sizeIncrement" >
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="alternatingRowColors" >
       <bool>true</bool>
      </property>
      <property name="selectionMode" >
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="horizontalScrollMode" >
       <enum>QAbstractItemView::ScrollPerItem</enum>
      </property>
      <property name="uniformRowHeights" >
       <bool>true</bool>
      </property>
      <property name="columnCount" >
       <number>7</number>
      </property>
      <column>
       <property name="text" >
        <string>1</string>
       </property>
      </column>
      <column>
       <property name="text" >
        <string>1</string>
       </property>
      </column>
      <column>
       <property name="text" >
        <string>2</string>
       </property>
      </column>
      <column>
       <property name="text" >
        <string>3</string>
       </property>
      </column>
      <column>
       <property name="text" >
        <string>4</string>
       </property>
      </column>
      <column>
       <property name="text" >
        <string>5</string>
       </property>
      </column>
      <column>
       <property name="text" >
        <string>6</string>
       </property>
      </column>
     </widget>
    </widget>
   </item>
   <item row="0" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>50</width>
         <height>30</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>&lt;h2>Directories&lt;/h2></string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>16</width>
         <height>30</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_5" >
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>15</height>
        </size>
       </property>
       <property name="text" >
        <string>&lt;h3>Restore Select&lt;/h3></string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>16</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="upButton" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>35</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>Up</string>
       </property>
       <property name="icon" >
        <iconset resource="../main.qrc" >:/images/up.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="markButton" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>35</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>Mark</string>
       </property>
       <property name="icon" >
        <iconset resource="../main.qrc" >:/images/check.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="unmarkButton" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>Unmark</string>
       </property>
       <property name="icon" >
        <iconset resource="../main.qrc" >:/images/unchecked.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>61</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>&lt;h2>Files&lt;/h2></string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>100</width>
         <height>30</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label_4" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>0</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Status:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="statusLine" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>0</hsizetype>
         <vsizetype>5</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Current Dir:</string>
       </property>
       <property name="buddy" >
        <cstring>lineEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit" >
       <property name="minimumSize" >
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>30</width>
         <height>32</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionMark" >
   <property name="icon" >
    <iconset resource="../main.qrc" >:/images/check.png</iconset>
   </property>
   <property name="text" >
    <string>Mark</string>
   </property>
  </action>
  <action name="actionUnMark" >
   <property name="icon" >
    <iconset resource="../main.qrc" >:/images/unchecked.png</iconset>
   </property>
   <property name="text" >
    <string>UnMark</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../main.qrc" />
 </resources>
 <connections/>
</ui>
